cockatrice/src/deckstats_interface.cpp

```diff
@@ -52,7 +52,8 @@ void DeckStatsInterface::getAnalyzeRequestData(DeckList *deck, QByteArray *data)
     urlQuery.addQueryItem("deck", deckWithoutTokens.writeToString_Plain());
     urlQuery.addQueryItem("decktitle", deck->getName());
     params.setQuery(urlQuery);
-    data->append(params.query(QUrl::EncodeReserved));
+    QString queryString = params.query(QUrl::EncodeReserved);
+    data->append(queryString.toUtf8());
```

Comment:
Can you not just say `data->append(param.query(QUrl::EncodeReserved).toUtf8());`?

Reply:
of course, it just becomes harder to read.

Reply:
I find this a bit more confusing as-is honestly. we're setting a temp variable to just use it next line

Reply:
I disagree, it breaks up the long statement, making it easier to interpret